packages/miniflare/test/plugins/cache/index.spec.ts

```diff
@@ -55,6 +55,18 @@ async function getControlStub(
 	return stub;
 }
 
+function sqlStmts(object: MiniflareDurableObjectControlStub) {
+	return {
+		getBlobIdByKey: async (key: string): Promise<string | undefined> => {
```

Comment:
Since this is duplicated in cache and kv tests, is it worth making it a shared helper? Not sure what your strategy in tests for that sort of thing is.